templates/items/item-sheet-2.hbs

```diff
@@ -0,0 +1,128 @@
+<form class="{{ cssClass }}" autocomplete="off">
+
+    {{!-- Header --}}
+    <header class="sheet-header">
+
+        {{!-- Image --}}
+        <div class="left">
+
+            <img class="item-image" src="{{ item.img }}" alt="{{ name.value }}" {{#if editable}}data-edit="edit"{{/if}}>
+
+        </div>
+
+        {{!-- Name & Subtitles --}}
+        <div class="middle identity-info">
+
+            {{!-- Name --}}
+            {{#if editable}}
+            {{ formInput name.field value=name.editable input=inputs.createTextInput
+                         classes="document-name uninput" }}
+            {{else}}
+            <div class="document-name">{{ name.value }}</div>
+            {{/if}}
+
+            {{!-- Type-Specific Subtitles --}}
+            <ul class="subtitles unlist">
+                {{~#each subtitles}}
+                {{#unless (and requiresIdentification @root.concealDetails)}}
+                {{#if @root.editable}}
+                {{#if field}}
+                <li {{#if classes}}class="{{ classes }}"{{/if}}>
+                    {{ formInput field value=value choices=choices blank=blank placeholder=placeholder localize=true }}
+                </li>
+                {{/if}}
+                {{else}}
+                {{#if label}}
+                <li {{#if classes}}class="{{ classes }}"{{/if}}><span>{{ label }}</span></li>
+                {{/if}}
+                {{/if}}
+                {{/unless}}
+                {{/each~}}
+            </ul>
+
+        </div>
+
+        {{!-- Weight, Price, Quantity, Level, & School --}}
+        <div class="right common-fields {{#if isPhysical}}physical{{/if}}">
+
+            {{#if isPhysical}}
+
+            {{!-- Quantity --}}
+            <div class="quantity">
```

Comment:
Having the numbers on the left for weight & cost but on the right for quantity feels a bit off. I wonder if we can find an icon for quantity and use that instead:

<img width="188" alt="Screenshot 2024-08-02 at 13 53 39" src="https://github.com/user-attachments/assets/711fc0f3-00c6-48a7-907a-84cb7ee12af8">

```html
<i class="fa-solid fa-cubes-stacked"></i> // for example
```

Reply:
I see what you mean, but the quantity icon isn't quite cutting it for me. I will try some things out on the forward branch.